tf_quant_finance/black_scholes/approximations/american_option.py

```diff
@@ -601,5 +681,63 @@ def _npdf(x):
   return tf.math.exp(-0.5 * x**2) / _SQRT_2_PI
 
 
+def _cbnd(dh, dk, rho):
+  """Compute values for the cumulative standard bivariate normal distribution,
+     specifically for use with Bjerksund-Stensland 2002 which always gives
+     the same value for rho.
+  #### References:
+  [1] Genz, A., Numerical Computation of Rectanglar Bivariate and Trivariate
+        Normal and t Probabilities, 2004
+    http://www.math.wsu.edu/faculty/genz/papers/bvnt.pdf"""
+
+  # Python code adapted from Fortran code and naming convention will align
+  # variables with the variables named in BVND function.
+  # http://www.math.wsu.edu/faculty/genz/software/fort77/tvpack.f
+
+  dtype = rho.dtype
+
+  # Gauss Legendre Points and Weights
+  w = tf.constant([0.01761400713915212, 0.04060142980038694, 0.06267204833410906,
```

Comment:
Please add a comment that unlike in the reference, for efficiency reasons on the GPU, we use 10 integration points regardless of `rho` values

Reply:
Updated.